docs/source/llms/langchain/notebooks/langchain-quickstart.ipynb

```diff
@@ -103,33 +103,39 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "/Users/benjamin.wilson/miniconda3/envs/mlflow-dev-env/lib/python3.8/site-packages/pydantic/_internal/_fields.py:128: UserWarning: Field \"model_server_url\" has conflict with protected namespace \"model_\".\n",
-      "\n",
-      "You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.\n",
-      "  warnings.warn(\n",
-      "/Users/benjamin.wilson/miniconda3/envs/mlflow-dev-env/lib/python3.8/site-packages/pydantic/_internal/_config.py:317: UserWarning: Valid config keys have changed in V2:\n",
-      "* 'schema_extra' has been renamed to 'json_schema_extra'\n",
-      "  warnings.warn(message, UserWarning)\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "import os\n",
     "\n",
     "from langchain.chains import LLMChain\n",
-    "from langchain.llms import OpenAI\n",
     "from langchain.prompts import PromptTemplate\n",
+    "from langchain_openai import OpenAI\n",
     "\n",
     "import mlflow\n",
     "\n",
     "assert \"OPENAI_API_KEY\" in os.environ, \"Please set the OPENAI_API_KEY environment variable.\""
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Notebook compatibility\n",
```

Comment:
nit: This cell might fit better in `Prerequisites` section that installs pip requirements.